torch/_functorch/_aot_autograd/schemas.py

```diff
@@ -310,13 +310,6 @@ def __post_init__(self):
         # sanity assert to make sure we don't leak memory
         assert is_fake(self.original_subclass)
 
-        # This saves the type of subclass nested structure to compare
-        # against runtime tangent inputs. We do wanna compute this at AOT
-        # time as it is invoked in hot-path
-        from .subclass_utils import get_types_for_subclass
-
-        self.subclass_type = get_types_for_subclass(self.original_subclass)
```

Comment:
Don't know why this still exist, i deleted it and repurposed the helper function for my use case lol